apps/verticalhome/test/unit/app_manager_test.js

```diff
@@ -8,7 +8,7 @@ require('/shared/js/component_utils.js');
 require('/shared/elements/gaia_confirm/script.js');
 require('/shared/js/homescreens/confirm_dialog_helper.js');
 
-suite('app_manager.js > ', function() {
+suite('more_apps_manager.js > ', function() {
```

Comment:
since you are renaming appManager to moreAppsManager, could you also rename the test itself?